water/modules/load_data.py

```diff
@@ -82,26 +82,27 @@ def load_pollution_dynamics():
     # Determine max and min pollution values and all scenario names
     max_pols = []
     min_pols = []
-    scenarios = []
+    pollution_nodes = []
```

Comment:
Is scenarios too vague/not descriptive?

I'm not sure that I like pollution nodes, it doesn't really clarify what the labels in this list correspond to. Maybe 'injection nodes' or 'origin nodes' would be better?